keywords/CouchbaseServer.py

```diff
@@ -250,3 +267,212 @@ def get_server_docs_with_prefix(self, url, bucket, prefix):
                 found_ids.append(row["$1"]["id"])
 
         return found_ids
+
+    def _wait_for_rebalance_complete(self, url):
+        """
+        Polls couchbase server tasks endpoint for any running rebalances.
+        Exits when no rebalances are in running state
+
+        /pools/default/tasks format:
+        [
+            {
+                "type": "rebalance",
+                "status": "running",
+                ...
+            }
+        ]
+        """
+
+        start = time.time()
+        while True:
+
+            if time.time() - start > keywords.constants.REBALANCE_TIMEOUT:
+                raise Exception("wait_for_rebalance_complete: TIMEOUT")
```

Comment:
what about adding some logging while it's stuck in this loop to make debugging easier?

"Waiting for rebalance.  Loop index: i.  Waiting for X seconds before retry"